gprofiler/profilers/perf.py

```diff
@@ -160,7 +160,10 @@ def merge_global_perfs(
 
     total_fp_samples = sum([sum(stacks.values()) for stacks in fp_perf.values()])
     total_dwarf_samples = sum([sum(stacks.values()) for stacks in dwarf_perf.values()])
-    fp_to_dwarf_sample_ratio = total_fp_samples / total_dwarf_samples
+    if total_dwarf_samples == 0:
```

Comment:
How did you encounter it now? You had a case when the fp perf produced samples but dwarf one did not?

Reply:
Yes I had `ZeroDivisionError` when I run gProfiler to profile single `docker run --rm python-listener` process (container built from tests/containers/python)

Reply:
kk